index.js

```diff
@@ -1608,6 +1608,18 @@
     return Bifunctor.methods.bimap(bifunctor)(f, g);
   }
 
+  //# mapLeft :: Bifunctor f => (a -> b, f a c) -> f b c
+  //.
+  //. Maps the given function over the first type argument of a Bifunctor
+  //.
+  //. ```javascript
+  //. > mapLeft(s => s.toUpperCase(), Tuple('foo', 'bar'))
+  //. Tuple('FOO', 'bar')
```

Comment:
I'm fond of using `Math.sqrt` in examples because it's short and doesn't require a lambda.

Reply:
⚡️